symfit/core/argument.py

```diff
@@ -46,6 +47,15 @@ def __init__(self, name=None, *args, **assumptions):
             self.name = name
         super(Argument, self).__init__()
 
+    def __getstate__(self):
+        state = super(Argument, self).__getstate__()
+        state.update(dict(
+            (slot, getattr(self, slot))
```

Comment:
Instead of this dict I would prefer to use the dict comprehension syntax, as it is more consistent with the rest of the project.